fury/molecular.py

```diff
@@ -0,0 +1,729 @@
+    colormode : string
+        Set the colormode for coloring the atoms. Two valid color modes -
+        * 'discrete': Atoms and bonds are colored using CPK coloring
+          convention.
+        * 'single': All atoms are colored with same color(grey) and all bonds
+          are colored with same color(dark grey).
+        * RGB tuple used for coloring the atoms when 'single' colormode is
+        selected: (150, 150, 150)
+        * RGB tuple used for coloring the bonds when 'single' colormode is
+        selected: (50, 50, 50)
+        * Default is 'discrete'.
+
+    atom_scale_factor : float
+        Scaling factor colormode='discrete',
+                               atom_scale_factor=0.3, bond_thickness=1,
+                               multipleBoto be applied to the atoms.
+        Default is 0.3.
+    bond_thickness : float
+        Used to manipulate the thickness of bonds (i.e. thickness of tubes
+        which are used to render bonds)
+        Default is 1.
+    multipleBonds : string
+        Set whether multiple tubes will be used to represent multiple
+        bonds. Two valid choices -
+        * 'On': multiple bonds (double, triple) will be shown by using
+          multiple tubes.
+        * 'Off': all bonds (single, double, triple) will be shown as single
+          bonds (i.e shown using one tube each).
+        Default is 'On'.
+
+    Returns
+    -------
+    molecule_actor : vtkActor
+        Actor created to render the ball and stick representation of the
+        molecule to be visualized.
+    """
+    bs_mapper = MoleculeMapper()
+    set_molecule_to_mapper(bs_mapper, molecule)
+    config_mapper_to_ball_stick(bs_mapper, colormode, atom_scale_factor,
+                                bond_thickness, multiple_bonds)
+    molecule_actor = vtk.vtkActor()
+    molecule_actor.SetMapper(bs_mapper)
+    make_molecularviz_aesthetic(molecule_actor)
+    return molecule_actor
+
+
+def molecular_stick_rep_actor(molecule, colormode='discrete',
+                              bond_thickness=1):
+    """Create an actor for stick molecular representation.
+
+    Parameters
+    ----------
+    molecule : Molecule object
+        The molecule to be rendered.
+    colormode : string
+        Set the colormode for coloring the bonds. Two valid color modes -
+        * 'discrete': Bonds are colored using CPK coloring convention.
+        * 'single': All bonds are colored with the same color (dark grey)
+        RGB tuple used for coloring the bonds when 'single' colormode is
+        selected: (50, 50, 50)
+        Default is 'discrete'.
+
+    bond_thickness : float
+        Used to manipulate the thickness of bonds (i.e. thickness of tubes
+        which are used to render bonds).
+        Default is 1.
+
+    Returns
+    -------
+    molecule_actor : vtkActor
+        Actor created to render the stick representation of the molecule to be
+        visualized.
+    """
+    mst_mapper = MoleculeMapper()
```

Comment:
@skoudoro  the MoleculeMapper class is used here to create the mapper for stick actor.